### PR TITLE
Hotfix: Update mimemagic to pull from github

### LINKED ISSUE #8: Problem with mimemagic on setting up the server
# Bug Report

Bug: Problem with mimemagic on setting up server

## Description

The problem is with `mimemagic` versions being yanked out so the specified version in `Gem.lock` is not working anymore.

### Steps to Reproduce

1. Clone a new repo
2. `docker-compose build`

### Expected Result

The expected result was...

Being able to run `docker-compose build` without any error

### Actual Results

The actual result was...

image

### PR DESCRIPTION
**Pull request by NeonNature**
# Story Title

[Problem with mimemagic on setting up the server](https://github.com/kuru-studio/kuru-studio-social-server/issues/8)

## Changes made

- Updated `mimemagic` on Gemfile to pull from GitHub directly

## How does the solution address the problem

This PR will solve the problem by pulling `mimemagic` directly from Github so it doesn't matter the versions are being yanked anymore.

## Linked issues

Resolves #8

### CODE REVIEW

**Comment by BosEriko** [comment]
We have a template.

**Comment by NeonNature** [comment]
> We have a template.

Updated with template

File: Gemfile [hunk 1]
```diff
@@ -48,3 +48,6 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# fix for the mimemagic no longer being supported
```
Comment (BosEriko) [inline Gemfile:52]: Can you link the issue here and capitalize the word "fix"?

**Comment by BosEriko** [comment]
LGTM after the comment has been resolved.

File: Gemfile (hunk 1 above)
Comment (NeonNature) [reply Gemfile:52]: link as in full url or just the issue number?

Comment (BosEriko) [reply Gemfile:52]: link

Comment (NeonNature) [reply Gemfile]: Updated

**Pull request merged**